python/ray/WebUI.ipynb

```diff
@@ -11,16 +11,528 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false
+    "collapsed": true
    },
    "outputs": [],
    "source": [
+    "import binascii\n",
+    "import json\n",
     "import os\n",
+    "import pandas as pd\n",
+    "import pandas.io.sql as psql\n",
+    "import qgrid \n",
     "import ray\n",
+    "import redis\n",
+    "import sys\n",
     "\n",
+    "from bokeh.plotting import figure, show\n",
+    "from bokeh.layouts import layout, widgetbox, row, gridplot\n",
+    "from bokeh.models import ColumnDataSource, HoverTool, Div, CustomJS, Range1d\n",
+    "from bokeh.models.widgets import Slider, Select, TextInput\n",
+    "from bokeh.io import curdoc, output_notebook, push_notebook\n",
+    "from pandas.io.json import json_normalize"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
     "redis_address = os.environ[\"REDIS_ADDRESS\"]\n",
     "ray.init(redis_address=redis_address)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "addr, port = ray.worker.global_worker.redis_address.split(\":\")\n",
+    "rc = redis.StrictRedis(host=addr, port=port)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Task and Actor Information "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Remote Function Information \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
```

Comment:
I'm not sure that the UI will be completely self explanatory to users. There's a lot of information here, and we may need to walk people through it. E.g., start with explaining what `ray.global_state.task_table()` does and then showing them how to put it in the dataframe. We may need text descriptions to go along with the code. On the other hand, people often don't read text, so that may not work. What do you think?